src/test/reload.test.ts

```diff
@@ -0,0 +1,137 @@
+import { Species } from "#app/enums/species.js";
+import { GameModes, getGameMode } from "#app/game-mode.js";
+import GameManager from "#test/utils/gameManager";
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import { SPLASH_ONLY } from "./utils/testUtils";
+import { Moves } from "#app/enums/moves.js";
+import { EnemyCommandPhase } from "#app/phases/enemy-command-phase.js";
+import { DamagePhase } from "#app/phases/damage-phase.js";
+
+describe("Reload", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+  });
+
+  it("should not have RNG inconsistencies in a Classic run", async () => {
+    await game.classicMode.startBattle();
+
+    const preReloadRngState = Phaser.Math.RND.state();
+
+    await game.reload.reloadSession();
+
+    const postReloadRngState = Phaser.Math.RND.state();
+
+    expect(preReloadRngState).toBe(postReloadRngState);
+  }, 20000);
+
+  it("should not have RNG inconsistencies after a biome switch", async () => {
+    game.override
+      .startingWave(15)
+      .battleType("single")
+      .startingLevel(100)
+      .enemyLevel(1000)
+      .disableTrainerWaves()
+      .moveset([Moves.KOWTOW_CLEAVE])
+      .enemyMoveset(SPLASH_ONLY);
+    await game.classicMode.startBattle();
+    game.scene.gameMode = getGameMode(GameModes.ENDLESS);
+
+    // Transition from Wave 15 to Wave 16 in order to trigger biome switch
+    game.move.select(Moves.KOWTOW_CLEAVE);
+    await game.phaseInterceptor.runFrom(EnemyCommandPhase).to(DamagePhase);
```

Comment:
```suggestion
    await game.phaseInterceptor.to("DamagePhase");
```
You don't need to use `runFrom()`.